Description: For an unkown reason, _() is undefine in keystone/exception.py and FTBFS
Author: Thomas Goirand <zigo@debian.org>
Forwarded: no
Last-Update: 2013-09-13

Index: keystone/keystone/exception.py
===================================================================
--- keystone.orig/keystone/exception.py	2014-03-28 14:35:45.000000000 +0800
+++ keystone/keystone/exception.py	2014-03-28 14:40:39.000000000 +0800
@@ -26,6 +26,8 @@
 # Tests use this to make exception message format errors fatal
 _FATAL_EXCEPTION_FORMAT_ERRORS = False
 
+def _(a):
+ return a
 
 class Error(Exception):
     """Base error class.
